axis_backend/apps/documents/tests/api/test_document_api.py

```
@@ -0,0 +1,559 @@
+        response = self.client_api.get(f'/api/documents/{doc.id}/check-expiry/')
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue(response.data['has_expiry'])
+        self.assertFalse(response.data['is_expired'])
+        self.assertEqual(response.data['days_until_expiry'], 15)
+        self.assertTrue(response.data['expires_soon'])
+
+    def test_expiring_soon(self):
+        """Test GET /api/documents/expiring-soon/ - Get expiring documents."""
+        # Create expiring document
+        doc = Document.objects.create(
```

Ruff (0.14.5)

Local variable `doc` is assigned to but never used

Remove assignment to unused variable `doc`

(F841)